pychromecast/controllers/plex.py

```diff
@@ -283,6 +284,14 @@ def next(self):
         """Send next command."""
         self._send_cmd({MESSAGE_TYPE: TYPE_NEXT})
 
+    def queue_next(self):
+        """Send QUEUE_NEXT as NEXT command."""
+        self.next()
+
+    def queue_prev(self):
+        """Send QUEUE_PREV as NEXT command."""
+        self.previous()
+
```

Comment:
Does Home Assistant really end up calling these functions; have you tested it?
The way I understand it, Home Assistant will still call `MediaController.queue_next`.